docs/docsite/rst/user_guide/complex_data_manipulation.rst

```diff
@@ -293,15 +293,13 @@ An example on how to use facts to find a host's data that meets condition X:
   vars:
     uptime_of_host_most_recently_rebooted: "{{ansible_play_hosts_all | map('extract', hostvars, 'ansible_uptime_seconds') | sort | first}}"
 
-
-Using an example from @zoradache on reddit, to show the 'uptime in days/hours/minutes' (assumes facts where gathered).
-https://www.reddit.com/r/ansible/comments/gj5a93/trying_to_get_uptime_from_seconds/fqj2qr3/
+Using an example on `reddit <https://www.reddit.com/r/ansible/comments/gj5a93/trying_to_get_uptime_from_seconds/fqj2qr3/>`_, to show the 'uptime in days/hours/minutes/seconds' (assumes facts where gathered).
```

Comment:
I would keep original author's name here.

Reply:
Please have a look at  https://github.com/ansible/ansible/pull/77009

[@samccann](https://github.com/samccann) [asked to not include it](https://github.com/ansible/ansible/pull/77009#discussion_r804918979)

We should probably not put someone's name in the docs.

Reply:
I question whether we even need to link to reddit from our docs.  My thoughts are the reddit links should just be removed.

Reply:
For me it doesn't matter if there is a link to reddit or not, I just wanted to correct the example:
not getting random ms data that are not in ansible_uptime_seconds.
Those example are copy pasted a lot, ...

Reply:
I wasn't sure about the reddit link either. I'd be happier to have it removed.